packages/cardpay-sdk/sdk/utils/module-utils.ts

```diff
@@ -18,7 +18,8 @@ export interface SetupArgs {
 export async function deployAndSetUpModule(
   ethersProvider: JsonRpcProvider,
   masterCopy: Contract,
-  setupArgs: SetupArgs
+  setupArgs: SetupArgs,
+  salt = 'cardstack-sp-deploy-module'
```

Comment:
If we're adding a salt, I think this should not default to a specific module, but be added by the calling code. SP here is I assume scheduled payments?

Reply:
I resolve this [here](https://github.com/cardstack/cardstack/pull/3603/commits/67151fb15e51078fa729fe1fe464c659e0e49cab)